Deeploy/Targets/Generic/Parsers.py

```
@@ -2555,3 +2627,170 @@ def parseNodeCtxt(self,
+        wellFormed = (kernel_shape is not None and len(node.outputs) == 1)
+        if wellFormed:
+            self.operatorRepresentation['strides'] = strides
+            self.operatorRepresentation['pads'] = pads
+            self.operatorRepresentation['kernel_shape'] = kernel_shape
+            self.operatorRepresentation['dilations'] = dilations
+            self.operatorRepresentation['group'] = group
+            self.operatorRepresentation['nodeName'] = node.name
+            self.operatorRepresentation['nodeOp'] = node.op
+        return wellFormed
+
+    def parseNodeCtxt(self, ctxt: NetworkContext, node: gs.Node, channels_first: bool = True):
```

Ruff (0.14.0)

Unused method argument: `channels_first`

(ARG002)